mypy.ini

```diff
@@ -0,0 +1,21 @@
+[mypy]
+pretty = True
+python_version = 3.8
+mypy_path = $MYPY_CONFIG_FILE_DIR/src:$MYPY_CONFIG_FILE_DIR/lib
+follow_imports = normal
+warn_redundant_casts = True
+warn_unused_ignores = True
+warn_unused_configs = True
+show_traceback = True
+show_error_codes = True
+
+# Enable PEP 420 packages
+namespace_packages = True
+
+# Have mypy locate packages without any __init__.py
+explicit_package_bases = True
+
+check_untyped_defs = True
+
+[mypy-ops.*]
+ignore_missing_imports = True
```

Comment:
Why? I think there should be a comment explaining this

Reply:
Added